extensions/wikia/CommunityPage/CommunityPageSpecialController.class.php

```diff
@@ -36,6 +36,7 @@ public function index() {
 			'heroImageUrl' => $this->getHeroImageUrl(),
 			'inviteFriendsText' => $this->msg( 'communitypage-invite-friends' )->plain(),
 			'headerWelcomeMsg' => $this->msg( 'communitypage-tasks-header-welcome', $wgWikiTopic ?? $wgSitename )->parse(),
+			'subheaderWelcomeMsg' => $this->msg( 'communitypage-subheader-welcome' )->text(),
 			'adminWelcomeMsg' => $this->msg( 'communitypage-admin-welcome-message' )->text(),
```

Comment:
let's remove `adminWelcomeMsg` from here and from .i18n.php file as well (`communitypage-admin-welcome-message`) 💃